app/uk/gov/hmrc/customsservicestatusfrontend/controllers/DashboardController.scala

```diff
@@ -16,17 +16,25 @@
 
 package uk.gov.hmrc.customsservicestatusfrontend.controllers
 
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.customsservicestatusfrontend.services.CustomsServiceStatusService
 import uk.gov.hmrc.customsservicestatusfrontend.views.html.DashboardPage
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
-import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+
 import javax.inject.{Inject, Singleton}
-import scala.concurrent.Future
+import scala.concurrent.ExecutionContext
 
 @Singleton
-class DashboardController @Inject() (mcc: MessagesControllerComponents, dashboardPage: DashboardPage) extends FrontendController(mcc) {
+class DashboardController @Inject() (
+  mcc:                         MessagesControllerComponents,
+  dashboardPage:               DashboardPage,
+  customsServiceStatusService: CustomsServiceStatusService
```

Comment:
Can we rename `CustomsServiceStatusService` to just `StatusService`? Understand why it's named the way it is given the name of the backend repo, but would prefer to avoid "ServiceStatusService".  StatusService is nice and easy to understand.